nestkernel/connection_manager.h

```diff
@@ -315,6 +318,8 @@ class ConnectionManager : public ManagerInterface
    */
   bool get_sort_connections_by_source() const;
 
+  bool get_use_compressed_spikes() const;
```

Comment:
I think code calling this method will be more readable if we call it just `use_compressed_spikes()`.

Reply:
ok